controllers/UIDialogController/index.js

```diff
@@ -20,8 +19,6 @@ import UIDialogTextInput from './UIDialogTextInput';
 const styles = StyleSheet.create({
     scrollContainer: {
         justifyContent: 'center',
-        paddingTop: UIConstant.normalContentOffset(),
```

Comment:
Top padding should be returned, otherwise all content will be sticked to the top